nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week2 homework"));
+
+app.get("/numbers/add", (req, res) => {
+    if (isNaN(req.query.first) && isNaN(req.query.second))
+        return res.status(406).json({ error: "Not a number" });
+
+    res.send(`Sum: ${req.query.first + req.query.second}`)
```

Comment:
You need to convert the strings (req.query.first) to numbers in order to be able to add them correctly.

Reply:
I`ve updated my request